pkg/client/cli/telepresence_test.go

```diff
@@ -1834,3 +1761,55 @@ func telepresenceContext(ctx context.Context, args ...string) (string, string) {
 
 	return strings.TrimSpace(stdout.String()), strings.TrimSpace(stderr.String())
 }
+
+func testContext(t testing.TB) context.Context {
+	return testContextWithConfig(t, nil)
+}
+
+func testContextWithConfig(t testing.TB, addConfig *client.Config) context.Context {
+	c := dlog.NewTestContext(t, false)
+	env, err := client.LoadEnv(c)
+	if err != nil {
+		t.Fatal(err)
+	}
+	c = client.WithEnv(c, env)
+
+	config := client.GetDefaultConfig(c)
+	config.LogLevels.UserDaemon = logrus.DebugLevel
+	config.LogLevels.RootDaemon = logrus.DebugLevel
+
+	to := &config.Timeouts
+	to.PrivateAgentInstall = 240 * time.Second
```

Comment:
If we have to bump all these timeouts in the tests, that makes me think maybe our default timeouts are too short?  While we need to overwrite config to test certain things, and some we need to overwrite for all tests (e.g. the registry), I think we should try to use as many defaults as possible in our tests so we are testing, as best as possible, what people get out of the box when they install Telepresence.  So perhaps bump the timeouts in `config.go` and remove setting the defaults here

Reply:
Not sure I agree. Our situation, where we use a sometimes extremely slow CI and not very fast cluster is, IMO out of the ordinary and this is a better safe than sorry timeout. For instance, bumping this default to 240s will leave the user hanging for four full minutes until telepresence gives up. That's quite exceptional and it's very likely to make the user unhappy when the problems are elsewhere (such as failure to pull the image, etc).

Reply:
yeah fair enough. Maybe it's worth trying to use more performant clusters in our CI in the future. My main concern is wanting to make sure our timeouts are good on average because we don't want a user's first use of telepresence to result in a timeout since that's not the best UX imo. But that's not exactly a problem for this PR and that should probably be a metric's based decision and not one based on a "hunch" which is all I have at this point.

Reply:
Totally agree with that. We should collect metrics for how long various things take. Perhaps we do already?

Reply:
I don't think we do, but it should be easy enough to sent a report every time we timeout w/ what the timeout was and what its value was.. so we can track that :) I'll make an issue just so we don't lose track of it.